### DISCUSSION TITLE
2.0.11

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#10811] [General]
*  Remove stale packaged COM registration during installation (solves #10782)
*  Remove usage of gai_strerrorA since it's not thread safe (solves #10690)
*  Resolve issue preventing some users from installing a 2.0.X version of the WSL package
*  Disable virtio9p while debugging issue preventing users from mounting drives

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.0.11'>2.0.11</a>.</em>

**Comment by ManoelMorais** [comment]
Good, you saved my problems
